src/components/SearchByToggle.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './SearchByToggle.scss';
+
+const SearchByToggle = ({onChange, value}) => {
+
+  // Assign the color to the button according to whether it is
+  // active or not
+  const getClassList = (criteria) => {
```

Comment:
I'm really happy that you're following BEM, for your sake, there is a package that make using it really easy within react. 

https://www.npmjs.com/package/bem-cn